### PR TITLE
cargo update

### PR DESCRIPTION
**Pull request by DiCesareDaniele**
Fixes #73

### CODE REVIEW

**Comment by matteo-dinoia** [comment]
Remove random line break and i'll merge if.

**Comment by DiCesareDaniele** [comment]
> Remove random line break and i'll merge if.

Which one?

File: README.md [hunk 1]
```diff
@@ -10,3 +10,11 @@ _Cargo.toml_
 wg_2024 = { git = "https://github.com/WGL-2024/WGL_repo_2024.git", features = ["serialize"] }
 ```
 if you don't want serde remove the features attribute
+
+Note that this repo is unstable and due to the volume of PR there will be a lot of breaking changes.  Thus it's important to update this dependency frequently. Cargo does not auto-update the dependencies
+> Once a `git` dependency has been added, Cargo will lock that dependency to the
+latest commit at the time. New commits will not be pulled down automatically
+once the lock is in place. However, they can be pulled down manually with
+`cargo update`.
```
Comment (matteo-dinoia) [inline README.md:18]:
```suggestion
> Once a `git` dependency has been added, Cargo will lock that dependency to the latest commit at the time. New commits will not be pulled down automatically once the lock is in place. However, they can be pulled down manually with `cargo update`.
```

**Review by matteo-dinoia — COMMENTED** [review]
This

**Pull request merged**